redesign/about.html

```diff
@@ -0,0 +1,183 @@
+    <meta name="author" content="@dwyl">
+    <meta name="keywords" content="do, what, you, love, dwyl, mvp, time, tracking, help, developers, code, frontend, backend, tooling, tools">
+    <meta name="robots" content="index, follow, noarchive">
+    <meta name="base" content="https://www.dwyl.com">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/dwyl.css">
+    <link rel="shortcut icon" href="/img/favicon.ico">
+    <title>dwyl</title>
+  </head>
+  <body>
+    <nav class="w-100 dwyl-bg-dark-gray tc white relative fixed">
+      <input type="checkbox" id="burger" class="absolute top-2 right-1 dn">
+      <label for="burger" class="dn-ns pointer absolute top-1 right-1">
+        <i class="fa fa-bars white dib f2"></i>
+      </label>
+      <ul class="overflow-hidden menu db-ns w-100 list tc pl0 pt3 mv0 f3 fw3 f5-ns">
+        <li class="absolute top-1 static-ns ph4 mh2 fw3 di-ns pt1 pb3 pv3-ns">
+          <a href="/" class="white link">
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="dwyl-dark-gray pt6 pb3 pb3-ns ph6-ns center shadow-4 tc">
+      <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w4 tc"/>
+      <h1 class="f1">Driven by Our Values</h1>
+    </section>
+
+    <section class="dwyl-dark-gray pt3 pb3 pb5-ns ph6-ns center tl w-60-l w-90">
+      <h2 class="dwyl-mint bb dwyl-b--gray pl3">dwyl</h2>
+      <p class="pl3 b mv1 f6">NOUN</p>
+      <div>
+        <p class="pl3 b dib mv1">Pronounciation:</p>
+        <p class="dib pl3 mv1 times-new-roman">/dwɪɫ/</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1">Origin:</p>
+        <p class="dib pl3 i mv0 times-new-roman">Do What You Love</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1 dib">1.</p>
+        <p class="dib pl2 i mv0 times-new-roman w-80 w-90-ns v-top pt1">
```

Comment:
Not sure this looks right on a new line with this alignment but also not a blocker.
<img width="584" alt="screen shot 2017-09-10 at 17 08 30" src="https://user-images.githubusercontent.com/4185328/30250690-cc08d7a2-964a-11e7-935e-6c0746f2b393.png">

Reply:
What view are you seeing it on a separate line? I did have an issue with this before but when I pushed up the view on mobile/ iPad and desktop was all on one line...

Reply:
This was a full desktop view on my macbook in Chrome. I can open an issue on it 👍